.cargo/config.toml

```diff
@@ -33,15 +33,17 @@ build-std = ["std", "panic_abort"]
 #build-std-features = ["panic_immediate_abort"] # Only necessary if building against ESP-IDF tag `v4.3.2` (the minimum supported version)
 
 [env]
+# Select ESP IDF version in embuild's format described here:
+# https://github.com/esp-rs/embuild/blob/v0.30.4/src/espidf.rs#L585
+#
 # Uncomment this to build against ESP-IDF 5.0 (master)
-#ESP_IDF_VERSION = { value = "master" }
-
+#ESP_IDF_VERSION = "master"
```

Comment:
Just to confirm: this short-hand syntax actually does work - this you have checked?

Reply:
* Yes, this time i checked it with a fresh shell
    ```
    $ cargo clean
    $ rm -rf .embuild
    $ env | grep IDF
    $ cargo build --release --target riscv32imc-esp-espidf
    [...]
        Finished release [optimized] target(s) in 2m 58s
    ```
* This is the format I initially got in contact with and this time I could also provide the [reference](https://doc.rust-lang.org/cargo/reference/config.html#env) from the Cargo docs. So no wild guess this time. ;-)